### PR TITLE
MINOR: Remove redundant throws in PipelineIR path

### PR DESCRIPTION
**Pull request by original-brownbear**
Trivial cleanup, preventing further dead code from creeping in :)

### CODE REVIEW

File: logstash-core/src/main/java/org/logstash/config/ir/graph/Graph.java [hunk 1]
```diff
@@ -330,14 +330,7 @@ public void validate() throws InvalidIRException {
     }
 
     public String toString() {
-        Stream<Edge> edgesToFormat;
-        try {
-            edgesToFormat = sortedEdges();
-        } catch (InvalidIRException e) {
-            // Even if this isn't a valid graph we still need to print it
-            edgesToFormat = edges.stream();
```
Comment (jakelandis) [inline logstash-core/src/main/java/org/logstash/config/ir/graph/Graph.java]: This looks like a functional change (not just clean up).  Is this passive?

Comment (original-brownbear) [reply logstash-core/src/main/java/org/logstash/config/ir/graph/Graph.java]: @jakelandis yea this is dead code, the exception is never thrown (the compiler now complains here after removing the redundant exceptions) :)

**Comment by jakelandis** [comment]
@original-brownbear - the code changes look fine, perhaps @andrewvc should do the final approval since even though the code here is dead it was _trying_ to do something and may want a follow up issue.

**Comment by original-brownbear** [comment]
@andrewvc thanks for taking a look :)

**Pull request closed without merging**